model/lib/mailbox/presentation_mailbox.dart

```diff
@@ -1,10 +1,13 @@
 import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
 import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox.dart';
 import 'package:jmap_dart_client/jmap/mail/mailbox/mailbox_rights.dart';
 import 'package:model/mailbox/select_mode.dart';
 
 class PresentationMailbox with EquatableMixin {
 
+  static final PresentationMailbox unifiedMailbox = PresentationMailbox(MailboxId(Id('unified')));
```

Comment:
what is this ?

Reply:
It is a Mailbox with id equal to null ((May be  define  `unified Mailbox`). To move the mailbox to the original `root`